mathics/builtin/natlang.py

```diff
@@ -119,6 +119,10 @@ def generator(doc):
         'Adjective': [nltk.corpus.wordnet.ADJ, nltk.corpus.wordnet.ADJ_SAT],
         'Adverb': [nltk.corpus.wordnet.ADV],
     }
+except LookupError:
+    print("warning: nltk dictionary not installed\n")    
```

Comment:
print isn't quite the correct way to handle this. We need to use `evaluation.message` to get the messaging appearing in all the Mathics interfaces. Furthermore, it looks like the parts of the code using these dicts don't expect them to be empty (the `ImportError` case is okay because `requires` prevents that code from running). Does this change fix the issue you're having?